doc/user_faq.rst

```diff
@@ -144,3 +144,10 @@ Then, when you want to include the packages from the rawhide repo, execute a DNF
 .. note::
 
     Installing rawhide packages onto a stable Fedora release system is generally discouraged as it leads to less tested combinations of installed packages. Please consider this step carefully.
+
+Starting with Fedora 40, I noticed repository metadata is synchronized much faster. What happened?
+===================================================================================================
+
+This is because filelists metadata is no longer downloaded by default. This change is associated with the Fedora system-wide `change <https://fedoraproject.org/wiki/Changes/DNFConditionalFilelists>`_, and the related `change <https://pagure.io/packaging-committee/pull-request/1321>`_ in the Fedora packaging guidelines policy, which specifies that packages should not rely on filepath dependencies.
```

Comment:
Since the difference between "should not" and "must not" is particularly important here, I would use "must not". Also, it is not exactly precise, since dependencies on some paths are allowed, so I would clarify it as "...which specifies that packages must not rely on filepath dependencies that would require filelists metadata." or something similar.